### ISSUE TITLE
allow multiple out endpoints to connect to one in endpoint

### ISSUE THREAD

**Issue by arlac77** [Kronos-Integration/kronos-step#4]
Let __in__ endpoints have more than one __out__ endpoint to connect to them
```
{
 "s1" : { "endpoints": { "out": "drain/in"}},
 "s2" : { "endpoints": { "out": "drain/in"}},
"drain" : { "endpoints" : { "in" : { ... } }  }
}
```
currently the __s2/out__ connection will fail

**Comment by darlenya** [comment]
I am just implementing this in the logger step.

**Comment by arlac77** [comment]
shouldn't it be a core feature ?

**Comment by arlac77** [comment]
closed with 0cfb552c02b87f4620437d81980babb10ef18770
